### PR TITLE
Update to Laravel 5.8

### PR DESCRIPTION
**Pull request by arcdigital**

### CODE REVIEW

**Review by lancepioch — APPROVED** [review]

File: resources/themes/pterodactyl/admin/nodes/view/allocation.blade.php [hunk 1]
```diff
@@ -159,7 +159,7 @@
                     </div>
                 </div>
                 <div class="modal-footer">
-                    {{{ csrf_field() }}}
+                    {{ csrf_field() }}
```
Comment (lancepioch) [inline resources/themes/pterodactyl/admin/nodes/view/allocation.blade.php]: Use csrf blade helper?

File: app/Providers/BroadcastServiceProvider.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Pterodactyl\Providers;
+namespace App\Providers;
```
Comment (lancepioch) [inline app/Providers/BroadcastServiceProvider.php]: Wrong namespace

Comment (arcdigital) [reply app/Providers/BroadcastServiceProvider.php]: hadn't finished the namespace change yet.

File: resources/themes/pterodactyl/admin/nodes/view/allocation.blade.php (hunk 1 above)
Comment (arcdigital) [reply resources/themes/pterodactyl/admin/nodes/view/allocation.blade.php]: you caught me before i did the bulk change, this was just to fix the unnecessary triple curly brace

**Review by DaneEveritt — CHANGES_REQUESTED** [review]

**Comment by DaneEveritt** [comment]
Let's roll with Laravel 6 in the develop branch once the dust settles a bit.

**Pull request closed without merging**